pkg/api/unversioned/well_known_labels.go

```diff
@@ -16,6 +16,6 @@ limitations under the License.
 
 package unversioned
 
-const LabelZoneFailureDomain = "failure-domain.alpha.kubernetes.io/zone"
-const LabelZoneRegion = "failure-domain.alpha.kubernetes.io/region"
+const LabelZoneFailureDomain = "failure-domain.beta.kubernetes.io/zone"
```

Comment:
Yes because I added a "Built-in node labels" section to node-selection/README.md and wanted to put the right thing there. Arguably this should be two PRs but it's small so I think it's easier to combine them.